src/Cli.Tests/Snapshots/UpdateEntityTests.TestUpdateEntityByAddingNewRelationship.verified.txt

```diff
@@ -5,7 +5,8 @@
   Runtime: {
     Rest: {
       Enabled: true,
-      Path: /
+      Path: /,
```

Comment:
Is `/` a valid path? [I acknowledge that this is different from what this PR aims to achieve, but thought of asking because of the change]

Reply:
Currently, yes, its a valid path.